tx.go

```diff
@@ -297,13 +299,19 @@ func IsValidTxID(txid []byte) bool {
 // Bytes encodes the transaction into a byte array.
 // See https://chainquery.com/bitcoin-cli/decoderawtransaction
 func (tx *Tx) Bytes() []byte {
-	return tx.toBytesHelper(0, nil)
+	return tx.toBytesHelper(0, nil, false)
```

Comment:
Here is a problem with the extended addition, we now have this flag being propagated everywhere

Reply:
Yes, this is propagated, but toBytesHelper() is a private function and I considered this to be an acceptable change.